### DISCUSSION TITLE
Installing pimcore 11 on macos throws error on locale

### DISCUSSION THREAD

**Discussion by agdrs** [pimcore/pimcore#15557] [Q&A / Support] [answered]
Pimcore v10.6 can be installed out-of-the-box, but v11 installation stops working here:

[ERROR] The following prerequisites failed
 * It is recommended to install UTF-8 locale, otherwise all CLI calls which use escapeshellarg() will strip multibyte characters (see https://packages.debian.org/en/stable/locales-all)

If i run the command `bin/console pimcore:system:requirements:check` it shows me an error for `locales-utf8`.
What can i do, to make it work without using docker?
Installing a v10.6 and upgrading to v11 does the trick, but that's not the way it has to be.
Any ideas or am i missing the point?

Thx and BR
Andreas

**Comment by agdrs** [comment]
Okay, the locale check is strictly dependent on linux OS, like debian.
The locales are named ['en.utf8', 'en_US.utf8', 'en_GB.utf8'], like the check in the Requirement.php says.
But for macOS you need to change the locales check to ['en.UTF-8', 'en_US.UTF-8', 'en_GB.UTF-8'].

macOS is not that unusual for developers, so it makes sense to have different checks for linux based systems and macOS.

**Reply by NiklasBr** [reply]
I suppose you could add those case variations in the check here:

https://github.com/pimcore/pimcore/blob/c21bbf42d04e9e6158aae58dc949b16ca1fac20a/lib/Tool/Requirements.php#L594-L599

**Reply by agdrs** [reply]
Hey @NiklasBr
thanks for your reply.

That's what i did and it worked for me.
But there are many others, so it would make sense to make those changes straight in the code provided to users. So every user in my team has to do manually and i don't know, if this check should be more flexible. And with the next update, the changes will be lost, even if this check will be run only once on install.

**Reply by NiklasBr** [reply, marked as answer]
It's actually quite easy to edit the file directly here on Github and submit a pull request, no need to pull the code, just edit directly in the web browser: https://docs.github.com/en/repositories/working-with-files/managing-files/editing-files#editing-files-in-another-users-repository

**Reply by agdrs** [reply]
Ah, now i understand what you mean.
Yeah thanks for the hint, i'll start a PR
